inspire/base/templates/header.html

```diff
@@ -128,14 +104,37 @@
   </nav>
 </section>
 
-{% if request.path  == "/search" %}
+{% if request.path  == "/search" or request.path | is_collection_path %}
+
+  {% if request.path  == "/search" %}
+    {% set collection_name = request.args.get('cc') %}
+  {% else %}
+    {% set collection_name = request.path | check_collection %}
+  {% endif %}
+
   <div id="collections">
     <div class="container">
-      {% set collection_name = request.args.get('cc') %}
-        <span id="{{collection_name | collection_select_current("Literature") }}">Literature</span>
-        <span id="{{collection_name | collection_select_current("Authors") }}">Authors</span>
-        <span id="{{collection_name | collection_select_current("Conferences") }}">Conferences</span>
-        <span id="{{collection_name | collection_select_current("Jobs") }}">Jobs</span>
+        <span id="{{collection_name | collection_select_current("Literature") }}">
```

Comment:
Not sure I understand why that filter is needed. To add to the id a suffix `-collection` ?